spirv_cross/src/wrapper.cpp

```diff
@@ -113,7 +146,6 @@ ScInternalResult sc_internal_compiler_get_entry_points(const ScInternalCompilerB
                 entry_points[i]->work_group_size_x = sc_entry_point.workgroup_size.x;
                 entry_points[i]->work_group_size_y = sc_entry_point.workgroup_size.y;
                 entry_points[i]->work_group_size_z = sc_entry_point.workgroup_size.z;
-                i++;
```

Comment:
I believe this was an error?

Reply:
@kvark Yes, good catch, thanks. The [loop was switched](https://github.com/grovesNL/spirv_cross/commit/c4c6a219a4b561df4893aebf498a2cbdef0c6d93#diff-7db03ad6f69bc7d45538d9707b908090L45) and it hasn't been tested yet.